cli/tests/install.bats

```diff
@@ -485,27 +485,26 @@ teardown() {
   assert_line --partial "The package 'hello-unfree' has an unfree license"
 }
 
+# This is also checking we can build an unfree package
 @test "catalog: 'flox install' warns about unfree packages" {
-  skip "will be fixed by https://github.com/flox/flox/issues/1507"
   "$FLOX_BIN" init
+  export  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello-unfree.json"
   run "$FLOX_BIN" install hello-unfree
   assert_success
   assert_line --partial "The package 'hello-unfree' has an unfree license"
 }
 
-@test "'flox install' fails to install unfree packages if forbidden" {
-  export FLOX_FEATURES_USE_CATALOG=false
+@test "catalog: 'flox install' warns about broken packages" {
+  skip "waiting for broken packages to be added to catalog"
   "$FLOX_BIN" init
-  tomlq --in-place -t '.options.allow.unfree = false' "$MANIFEST_PATH"
-
-  run "$FLOX_BIN" install hello-unfree
-  assert_failure
-  assert_line --partial "The package 'hello-unfree' has an unfree license."
-  assert_output --partial "'options.allow.unfree = true'"
+  run "$FLOX_BIN" install TODO
+  assert_success
+  assert_line --partial "The package 'TODO' is marked as broken, it may not behave as expected during runtime"
 }
 
-@test "catalog: 'flox install' fails to install unfree packages if forbidden" {
```

Comment:
> Note that with pkgdb, if a package failed to install because it was not
> allowed, a helpful error was printed suggesting a modification to
> options.allow. This is no longer possible since resolution failures do
> not currently include enough information to determine if the failure was
> due to an options.allow setting. The corresponding copied but skipped
> integration tests were dropped instead of re-enabling.

Should we capture re-enabling these tests in (or as a result of) https://github.com/flox/catalog-server/issues/106?

Reply:
Left a comment